src/Microsoft.Identity.Client/Instance/B2CAuthority.cs

```diff
@@ -37,7 +37,8 @@ internal class B2CAuthority : AadAuthority
     {
         public const string Prefix = "tfp"; // The http path of B2C authority looks like "/tfp/<your_tenant_name>/..."
         public const string B2CCanonicalAuthorityTemplate = "https://{0}/{1}/{2}/{3}/";
-        public const string B2CTrustedHost = "b2clogin.com";
+        // public const string B2CTrustedHost = "b2clogin.com";
+        private string[] B2CTrustedHosts = { "b2clogin.com", "b2cLogin.cn" };
```

Comment:
is this case sensitive? b2cLogin.cn vs b2clogin.cn

Reply:
the comparison is set to ignore case... however I suppose this looks better if we stick to the same casing.

---
In reply to: [257044223](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/884#discussion_r257044223) [](ancestors = 257044223)

Reply:
cool. Also, should this be readonly? Also, any reason to not do a HashSet<string> like we do in AadAuthority (now that we have more than 2)?

---
In reply to: [257403526](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/884#discussion_r257403526) [](ancestors = 257403526,257044223)